src/main/java/org/quiltmc/quiltmappings/loom/QuiltMappingsOnLoomPlugin.java

```diff
@@ -110,6 +116,10 @@ public void visit(MappingVisitor mappingVisitor) throws IOException {
             if (!intermediaryToQm.exists()) {
                 List<File> quiltMappings = new ArrayList<>(project.getConfigurations().detachedConfiguration(project.getDependencies().create(this.quiltMappings)).resolve());
 
+                if (quiltMappings.size() < 2) {
+                    throw new UnsupportedOperationException("The Quilt Mappings build used here is likely to be a post-Intermediary publications build, which are incompatible with Quilt Mappings on Loom!");
```

Comment:
I would add examples of what to put for both layered and non layered mappings here

Reply:
Maybe linking to QTM's examples would work?

Reply:
Sure!